extensions/interactions/EndExploration/directives/end-exploration-interaction.directive.html

```diff
@@ -1,3 +1,11 @@
-<div ng-if="$ctrl.isInEditorPage" style="color: red; font-size: 0.8em; margin-top: 20px;">
+<div ng-if="$ctrl.isInEditorPage" class="oppia-error-message-text">
   <span><[$ctrl.errorMessage]></span>
 </div>
+
+<style>
+  oppia-interactive-end-exploration .oppia-error-message-text {
+    color: red;
```

Comment:
Use HEX value for colors (for cross-browser consistency!)

Reply:
Done.